tasks/javascript.js

```diff
@@ -113,6 +115,17 @@ gulp.task('javascript:lint', function () {
 
 // karma unit tests in local library only
 gulp.task('javascript:unit', function () {
+  var reporters = cliArgs.karmareporter;
+  if (reporters.constructor === Array) {
+    reporters = Array;
+  }
+  else if (typeof reporters === 'string' &&
+      reporters !== karma.yargsOption.value.default) {
+    reporters = [reporters];
+  }
+  else {
+    reporters = [];
```

Comment:
Based on the preceding `if` clause I am presuming that an empty value indicates default reporter.

Is it a use case that there be no reporter? In the case of jsHint this case `throws`. We should at least be consistent.